css-view-transitions-2/Overview.bs

```diff
@@ -631,17 +631,18 @@ div.box {
 	are extended to support the following syntax:
 
 	<pre class=prod>
-		::view-transition-group(<<pt-name-selector>><<pt-class-selector>>)
-		::view-transition-image-pair(<<pt-name-selector>><<pt-class-selector>>)
-		::view-transition-old(<<pt-name-selector>><<pt-class-selector>>)
-		::view-transition-new(<<pt-name-selector>><<pt-class-selector>>)
+		::view-transition-group(<<pt-name-and-class-selector>>)
+		::view-transition-image-pair(<<pt-name-and-class-selector>>)
+		::view-transition-old(<<pt-name-and-class-selector>>)
+		::view-transition-new(<<pt-name-and-class-selector>>)
 	</pre>
 
 	where <<pt-name-selector>> works as previously defined, and
-	<<pt-class-selector>> has the following syntax definition:
+	<<pt-name-and-class-selector>> has the following syntax definition:
 
 	<pre class=prod>
-		<dfn>&lt;pt-class-selector></dfn> = ['.' <<custom-ident>>]*
+		<dfn>&lt;pt-name-and-class-selector></dfn> = <<pt-name-selector>><<pt-class-selector>>? | <<pt-class-selector>>
```

Comment:
And here "?" implies zero or more class selectors.

Reply:
Yes, zero or one `<<pt-class-selector>>`, which would be one or more (because it has +)